src/cmd-osbuild

```diff
@@ -242,6 +246,7 @@ main() {
         "cmd-buildextend-qemu") platforms=(qemu);;
         "cmd-buildextend-qemu-secex") platforms=(qemu-secex);;
         "cmd-buildextend-secex") platforms=(qemu-secex);;
+	"cmd-buildextend-live") platforms=(live-iso);;
```

Comment:
we'd need a corresponding entry in the `Perform postprocessing` case statement in this file to handle the multiple artifacts that are being generated (i.e. not just the ISO)